src/Sulu/Bundle/MediaBundle/Resources/config/image-formats.xml

```diff
@@ -4,27 +4,59 @@
          xsi:schemaLocation="http://schemas.sulu.io/media/formats http://schemas.sulu.io/media/formats-1.1.xsd">
 
     <format key="400x400">
+        <meta>
+            <title lang="en">Contact avatar (Sulu)</title>
+            <title lang="de">Kontaktavatar (Sulu)</title>
```

Comment:
I agree. However, we'd also have to translate for example the templates to all languages:
```
<meta>
    <title lang="de">Standard</title>
    <title lang="en">Default</title>
</meta>
```